### PR TITLE
Added showHelp and showSettings inputs to tiles

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Issue: #1109 
Docs: Included in PR

### CODE REVIEW

File: js/sky/src/tiles/docs/demo.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ When used on forms, it automatically adjusts the background color on the form an
  - `bb-tile-help-click` &mdash; A function to call when the user clicks the help button (indicated by a question-circle icon) in the tile header. If not specified, the help button is not displayed.
  - `bb-tile-settings-click` &mdash; A function to call when the user clicks the settings button (indicated by a wrench icon) in the tile header. If not specified, the settings button is not displayed.
  - `bb-tile-collapsed` &mdash; (optional) binds to the collapsed state of the tile so that the tile can respond to user setting collapsed state.
+ - `bb-tile-show-help` &mdash; (optional) Indicates whether to display a help button in the tile's header. To display the button, you must also supply a bb-tile-help-click function. (Default: true)
```
Comment (blackbaud-johnly) [inline js/sky/src/tiles/docs/demo.md]: I think "bb-tile-help-click" should be a code snippet.

File: js/sky/src/tiles/docs/demo.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ When used on forms, it automatically adjusts the background color on the form an
...
+ - `bb-tile-show-settings` &mdash; (optional) Indicates whether to display a settings button in the tile's header. To display the button, you must also supply a bb-tile-settings-click function. (Default: true)
```
Comment (blackbaud-johnly) [inline js/sky/src/tiles/docs/demo.md]: I think "bb-tile-settings-click" should be a code snippet.

File: js/sky/src/tiles/docs/demo.md (hunk 1 above)
Comment (blackbaud-conorwright) [reply js/sky/src/tiles/docs/demo.md]: done

File: js/sky/src/tiles/docs/demo.md (hunk 2 above)
Comment (blackbaud-conorwright) [reply js/sky/src/tiles/docs/demo.md]: done

**Review by blackbaud-johnly — APPROVED** [review]
Can't speak to the code, but doc changes look good to me.

File: js/sky/src/tiles/docs/demo.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ When used on forms, it automatically adjusts the background color on the form an
  - `bb-tile-help-click` &mdash; A function to call when the user clicks the help button (indicated by a question-circle icon) in the tile header. If not specified, the help button is not displayed.
  - `bb-tile-settings-click` &mdash; A function to call when the user clicks the settings button (indicated by a wrench icon) in the tile header. If not specified, the settings button is not displayed.
  - `bb-tile-collapsed` &mdash; (optional) binds to the collapsed state of the tile so that the tile can respond to user setting collapsed state.
+ - `bb-tile-show-help` &mdash; (optional) Indicates whether to display a help button in the tile's header. To display the button, you must also supply a `bb-tile-help-click` function. (Default: true)
```
Comment (blackbaud-johnly) [inline js/sky/src/tiles/docs/demo.md]: Sorry, but I have one more docs tweak: The style for defaults is to italicize them and format them as code snippets. So "true" here and in the next item should be code snippets that are italicized.

Comment (blackbaud-conorwright) [reply js/sky/src/tiles/docs/demo.md]: Done I believe

File: js/sky/templates/tiles/tile.html [hunk 4]
```diff
@@ -9,6 +9,7 @@ <h2 class="bb-tile-header">{{bbTile.tileHeader}}</h2>
                     <button ng-if="bbTile.hasHelp"
                         class="bb-tile-help fa fa-question-circle"
                         ng-click="$event.stopPropagation();bbTile.bbTileHelpClick();"
+                        type="button"
```
Comment (Blackbaud-SteveBrush) [inline js/sky/templates/tiles/tile.html:12]: Good catch!

**Review by Blackbaud-SteveBrush — APPROVED** [review]
LGTM

**Pull request merged**